### DISCUSSION TITLE
doesn't work here. Big Sur 11.3.1, v2 beta9

### DISCUSSION THREAD

**Discussion by jacobax** [noah-nuebling/mac-mouse-fix#109] [Questions] [answered]
already put in applications folder and accessiblity enabled. 
and "enable Mac mouse fix" automatically disabled once the windows is closed and reopen. with it enabled or not, doesn't function at all.
I first used v1.0.4 and it works perfectly and I don't know why it suddenly didn't, so I tried reinstalling and different versions.
what did I miss? help please and thank you for your great job
<img width="857" alt="Screen Shot 2021-05-13 at 14 46 18" src="https://user-images.githubusercontent.com/23228895/118090944-c39e6400-b3fc-11eb-81cc-0999f1c7847f.png">

**Comment by noah-nuebling** [comment, marked as answer]
Hey @Jacobax, sorry for the late reply.

For one other person who had the same issue it was cause by the permissions on their LaunchAgents folder being set to „read only“ somehow. For another person it magically worked with the latest Beta, I‘m not sure why.

So I‘d suggest you try:
1. Going to ~/Library/LaunchAgents > Right click > Show Info and then making sure the permission setting at the bottom are set to read and write 
2. Trying the latest Beta (11)

Let me know if that helps and if you have any more questions.

**Reply by jacobax** [reply]
thanks for the reply.
I just try both 2 suggestions, it works great.
I have updated the os from 11.3.1 to 11.4, so I don't know what exactly makes it right.

**Reply by noah-nuebling** [reply]
Ah that‘s good to hear! Thanks for letting me know.
